content/outcomes/application-development-how-we-work/_index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: How Tanzu Labs App Services Works and Why
```

Comment:
Is _App Services_ even an entity we need to surface externally? In the past we've done a poor job at letting our internal org structure show up in how we engage with customers - are we repeating those mistakes by not simply saying _VMware Tanzu Labs_?

Reply:
Good point.

Reply:
Updated